pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -97,6 +100,10 @@ type TektonConfigStatus struct {
 	// The version of the installed release
 	// +optional
 	Version string `json:"version,omitempty"`
+
+	// The current installer set name
+	// +optional
+	TektonInstallerSet map[string]string `json:"tektonInstallerSets,omitempty"`
```

Comment:
In my opinion, a list of names of TektonInstallerSets directly managed by TektonConfig is of low value. 
In addition, as  rbac is a platform specific operation let us keep the effects away from this common area.

As there will be only one InstallerSet for RBAC, we can keep the name constant and we do not have to record that as a status field.

Reply:
Added status field to indicate creation of rbac resources are enabled or disabled

/cc @sm43

Reply:
for TektonPipelines, there were multiple installerSet created and keeping hardcoded name wasn't a good option
and status also works as an check whether we have enabled that part or not
like if the name of `installerSet` in status doesn't exist and `createRbacResource` is `false` then we can just save the delete api call which we would be doing on each reconcile

Reply:
and if pruner is going to use installerSet as ownerRef then this would be visible in both platform
otherwise the field will not be visible on k8s